website/manual.md

```diff
@@ -234,7 +234,7 @@ Environment variables: `DENO_BUILD_MODE`, `DENO_BUILD_PATH`, `DENO_BUILD_ARGS`,
 To get an exact reference of deno's runtime API, run the following in the
 command line:
 
-```shellsession
+```shell
 $ deno types
```

Comment:
Thanks for the PR. I'm sad to hear that shellsession isn't a valid language....

This one, isn't valid shell script because of the $. I think we should either remove the $ or remove the highlighting. Probably the latter.

Reply:
`shell`  means "shell session", can contain `$`, but `bash`/`sh` can't.

    ```shell
    $ deno types
    ```

    ```bash
    deno types
    ```

Reply:
ah, got it. Thanks!